tests/test-cases/Group24-Multi-VCH/24-01-Multi-VCH-PS.robot

```diff
@@ -0,0 +1,54 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 6-04 - Verify vic-machine create basic use cases
+Resource  ../../resources/Util.robot
+Test Teardown  Extra Cleanup
+
+*** Keywords ***
+Extra Cleanup
+      Manually Cleanup VCH  %{GROUP-14-MULTI-PS-VCH}
+      Cleanup VIC Appliance On Test Server
```

Comment:
I would make this one first, and add Run Keyword and Continue on Failure.  So that if it has a problem cleaning up the first one it will still attempt to cleanup the 2nd one